understanding/sharding.md

```diff
@@ -11,20 +11,168 @@ You do not need to worry about sharding until your bot hits around 2,400 guilds.
 There are additional difficulties when sharding a bot that add complexity to your code \(one of the reasons you shouldn't shard too early\).
 
 * Collections do not cache data from all shards, so you can't grab data from a guild in another shard easily.
-* In order to do anything across shards you need to worry about using `broadcastEval` and such \(tutorial coming soon!\).
+* In order to do anything across shards you need to worry about using `broadcastEval` and such \(Examples and explanation below\).
 * Sharded bots often gain very marginal performance increase and might even use _more_ memory due to using more node processes.
 * If you're using any sort of database or connection, multiple shards may cause issues with multiple processes connecting to a single end point.
 
-## Example Sharding Code
+## Example Sharding Manager Code
 
 ```javascript
 /*
     The following code goes into it's own file, and you run this file
     instead of your main bot file.
 */
 
-const Discord = require('discord.js');
-const Manager = new Discord.ShardingManager('./YOUR_BOT_FILE_NAME.js');
-Manager.spawn(2); // This example will spawn 2 shards (5,000 guilds);
+// Include discord.js ShardingManger
+const { ShardingManager } = require('discord.js');
+
+// Create your ShardingManger instance
+const manager = new ShardingManager('./YOUR_BOT_FILE_NAME.js', {
+    // for ShardingManager options see:
+    // https://discord.js.org/#/docs/main/master/class/ShardingManager
```

Comment:
This should target `stable`, as not everyone targets master.
```suggestion
    // https://discord.js.org/#/docs/main/stable/class/ShardingManager
```

Reply:
Same thing for all other instances.

Reply:
Should probably just be "v12" to stay consistent with my other pr and to maintain correct links through future versions.